src/main/resources/resources/logisim/strings/util/util_fr.properties

```diff
@@ -85,7 +85,10 @@ questaValidationIoException = Une erreur d'E/S s'est produite lors de la validat
 variableDoubleUnderscore = Erreur : symboles "_" concaténés détectés!\n
 variableEndsWithUndescore = Erreur : le nom se termine par le symbole "_"!\n
 variableInvalidCharacters = Erreur : caractères non valides détectés!\n
+variableIllegalCharacter = Erreur : Le caract�re "%s" n'est pas autoris�.\n
```

Comment:
There seems to be an encoding problem with the accented letters.

Reply:
Yeah, I had inserted the text as UTF-8 by accident